config/ready_test.go

```diff
@@ -641,26 +641,27 @@ func TestDiffConfigReady(t *testing.T) {
 }
 
 func TestReportConfigReady(t *testing.T) {
-	os.Setenv("GITHUB_EVENT_NAME", "pull_request")
-	os.Setenv("GITHUB_EVENT_PATH", filepath.Join(rootTestdataDir(t), "config", "event_pull_request_opened.json"))
-	os.Setenv("GITHUB_REF", "refs/pull/4/merge")
-	mg := mockedGh(t)
+	t.Setenv("GITHUB_EVENT_NAME", "pull_request")
+	t.Setenv("GITHUB_EVENT_PATH", filepath.Join(rootTestdataDir(t), "config", "event_pull_request_opened.json"))
+	t.Setenv("GITHUB_REF", "refs/pull/4/merge")
+	t.Setenv("GITHUB_TOKEN", "token")
```

Comment:
Switched from os.SetEnv to t.SetEnv. The reason is to eliminate implicit dependencies between tests.

Also, I found it when I was testing locally, but GITHUB_TOKEN is required for mockedGh, so I also set it in env